rpc/backend/tracing.go

```diff
@@ -31,12 +31,16 @@ import (
 // and returns them as a JSON object.
 func (b *Backend) TraceTransaction(hash common.Hash, config *evmtypes.TraceConfig) (interface{}, error) {
 	// Get transaction by hash
-	transaction, err := b.GetTxByEthHash(hash)
+	transaction, enblock, err := b.GetTxByEthHash(hash)
 	if err != nil {
 		b.logger.Debug("tx not found", "hash", hash)
 		return nil, err
 	}
 
+	if enblock {
```

Comment:
what is connection between `endblock` flag and cross chain txs?

Reply:
it determines if the tx we are feting is from the end block tx we applied as part of omni or a normal evm tx made by a real tx.